libvirtd.sh

```diff
@@ -16,6 +16,9 @@ mkdir /dev.container && {
   keep pts
   mount --rbind /dev/pts/ptmx /dev/ptmx
   # Use the container /dev/kvm if available
+  # Replace inherited permissions from host with ours, as we have a different passwd file
+  # we may use different uids, e.g. kvm on the host is not kvm within the container.
+  [[ -e /dev.container/kvm ]] && chown root:kvm /dev.container/kvm && chmod 660 /dev.container/kvm
```

Comment:
`/dev/kvm` is usually chmod-ed 666, which is probably a good idea here too, especially if we ever switch to non-root libvirt.

Reply:
it is? On my test machine it was set to 660 by default.
Do you have a source for that?

Reply:
50-udev-defaults from systemd sets this on modern distros, and it's configurable: https://github.com/systemd/systemd/blob/master/rules.d/50-udev-default.rules.in#L86

On Fedora it's `0666` and RHEL too IIRC but I didn't confirm

Reply:
But why exactly does then a "kvm" group exist? Isn't that group than completely redundant as everybody has access anyway?
Couldn't the file not just be owned by root:root with 0666 as permission??

Reply:
I don't claim to understand the full implications of the various permission bits, but please keep in mind that SELinux is part of the picture too:

```
$ ls -lZ /dev/kvm 
crw-rw-rw-. 1 root kvm system_u:object_r:kvm_device_t:s0 10, 232 Jun 26 12:28 /dev/kvm
```

Reply:
'kvm' group predates widespread 0666 permissions on /dev/kvm. Also for a long time distro qemu packages were adding udev rules to set the 0666 permissions (before those rules were moved to systemd), so the host default was still 0660 unless qemu was installed. I believe the kvm group is just a historical artifact at least for distros that are setting 0666 permissions.

Reply:
Apparently the default by now is 0666: https://github.com/systemd/systemd/blob/master/meson_options.txt#L218-L219 and https://github.com/systemd/systemd/blob/master/meson.build#L797-L799